app/controllers/api/notifications_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Api
+  class NotificationsController < BaseController
+    def notifications_search_conditions
+      {:user_id => @auth_user_obj.id}
+    end
+
+    def find_notifications(id)
+      @auth_user_obj.notification_recipients.find(id)
+    end
+
+    def mark_as_seen_resource_notifications(type, id = nil, _data = nil)
```

Comment:
if there is no identifier, should we check that this type/id notification is for @auth_user_obj here ?